DataFormats/ForwardDetId/interface/FastTimeDetId.h

```diff
@@ -14,7 +14,7 @@ class FastTimeDetId : public DetId {
   static const int kFastTimeCellPhiMask     = 0x3FF;
   static const int kFastTimeZsideMask       = 0x100000;
```

Comment:
@bsunanda Since you made everything else with consistent notation could you make the Z-side done in the same way (bit mask + offset)?